.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     rev: v0.3.1
     hooks:
       - id: absolufy-imports
-        name: absolufy-imports
+        files: ^xarray/
```

Comment:
you'll need both lines for this to work 😉

Reply:
Ok, I just copy/pasted the pandas solution. What's the difference?

Reply:
Ah you're right, just `id` is enough, sorry for the noise